public/js/components/Breakpoints.js

```diff
@@ -73,10 +58,21 @@ const Breakpoints = React.createClass({
   }
 });
 
+function _getBreakpoints(state) {
+  return getBreakpoints(state).map(breakpoint => {
+    const source = getSource(state, breakpoint.getIn(["location", "actor"]));
+    const isCurrentlyPaused = isCurrentlyPausedAtBreakpoint(state, breakpoint);
+    const locationId = makeLocationId(breakpoint.get("location").toJS());
+    return breakpoint.setIn(["location", "source"], source)
+                     .set("locationId", locationId)
+                     .set("isCurrentlyPaused", isCurrentlyPaused);
+  });
+}
```

Comment:
I think you're right that this selector makes more sense inside of the Breakpoints component. Good call